src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -119,8 +121,25 @@ bool mpu6000ReadRegister(uint8_t reg, uint8_t length, uint8_t *data)
     return true;
 }
 
-void mpu6000SpiGyroInit(uint8_t lpf)
+void mpu6000SpiGyroInit(gyro_t *gyro, uint8_t lpf)
 {
+    uint16_t intPeriod;
+    switch(lpf) {
+        case 0:
+            intPeriod = PERIOD_HZ(8000);
```

Comment:
my initial thought, was that users are used to talking about values like 8k/4k, etc.  hence gyro_sample_hz and the use of macros.